test/AutoRest.TestServer.Tests/BodyAndPath.cs

```diff
@@ -17,7 +17,7 @@ public void BodyParameterIsLast()
             ParameterInfo[] parameters = typeof(BodyAndPathClient).GetMethod("Create").GetParameters();
             Assert.AreEqual(typeof(string), parameters[0].ParameterType);
             Assert.AreEqual(typeof(RequestContent), parameters[1].ParameterType);
-            Assert.AreEqual(typeof(CancellationToken), parameters[2].ParameterType);
+            Assert.AreEqual(typeof(RequestOptions), parameters[2].ParameterType);
```

Comment:
is this LLC test class? Might be good to add a suffix to the testt name.

Reply:
No, https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.Experimental/src/RequestOptions.cs

Reply:
Yes - It is a test - my bad for not adding the suffix.

Reply:
I'll fix it.

Reply:
Resolved.